### PR TITLE
Update changelog for metadata

### PR DESCRIPTION
**Pull request by jbajic**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Future
+
+### Major Features and Improvements
+
+* Added bolt notifications to inform user about results ot to give useful tips. [#285](https://github.com/memgraph/memgraph/pull/285)
```
Comment (antaljanosbenjamin) [inline docs/changelog.md:11]:
```suggestion
* Added Bolt notifications to inform user about results or to give useful tips. [#285](https://github.com/memgraph/memgraph/pull/285)
```

File: docs/changelog.md [hunk 2]
```diff
@@ -13,114 +13,170 @@ sidebar_label: Changelog
 * Allow duration values to be used as weights in the [Weighted Shortest
   Path](https://memgraph.com/docs/memgraph/reference-guide/graph-algorithms#weighted-shortest-path)
   query. [#278](https://github.com/memgraph/memgraph/pull/278)
-* Fix linkage error when `mgp_local_time_get_minute` is used. [#273](https://github.com/memgraph/memgraph/pull/273)
+* Fix linkage error when `mgp_local_time_get_minute` is used.
+  [#273](https://github.com/memgraph/memgraph/pull/273)
+* Added Bolt notifications to inform the user about results or to give useful
+  tips. [#285](https://github.com/memgraph/memgraph/pull/285)
+* Added execution statistics to inform user on how much object were affected.
```
Comment (g-despot) [inline docs/changelog.md:20]: It should be either:
-"how many objects were affected" or
-"how objects were affected".

File: docs/changelog.md [hunk 3]
```diff
@@ -165,31 +227,33 @@ sidebar_label: Changelog
...
+  transaction), ASYNC (MAIN doesn't care if data is replicated), SYNC WITH
   TIMEOUT (MAIN will wait for REPLICAS within the given timeout period, after
   timout, replication isn't aborted but the replication demotes the REPLICA to
   the ASYNC mode).
 * Added support for query type deduction. Possible query types are `r` (read),
   `w` (write), `rw` (read-write). The query type is returned as a part of the
   summary.
 * Improved logging capabilities by introducing granular logging levels. Added
-  new flag, `--log-level`, which specifies the minimum log level that will be
+  new flag, `--log-level` , which specifies the minimum log level that will be
```
Comment (kostasrim) [inline docs/changelog.md:253]:
```suggestion
  new flag, `--log-level`, which specifies the minimum log level that will be
```

File: docs/changelog.md [hunk 4]
```diff
@@ -165,31 +227,33 @@ sidebar_label: Changelog
...
   printed. E.g., it's possible to print incoming queries or Bolt server states.
 * Added ability to lock the storage data directory by executing the `LOCK DATA
-  DIRECTORY;` query which delays the deletion of the files contained in the
+  DIRECTORY; ` query which delays the deletion of the files contained in the
```
Comment (kostasrim) [inline docs/changelog.md:256]:
```suggestion
  DIRECTORY` query which delays the deletion of the files contained in the
```

File: docs/changelog.md [hunk 5]
```diff
@@ -221,9 +287,9 @@ sidebar_label: Changelog
 ### Bug Fixes and Other Changes
 
 * Fixed Cypher `ID` function `Null` handling. When the `ID` function receives
-  `Null`, it will also return `Null`.
-* Fixed bug that caused random crashes in SSL communication on platforms
-  that use older versions of OpenSSL (< 1.1) by adding proper multi-threading
+  `Null` , it will also return `Null` .
```
Comment (kostasrim) [inline docs/changelog.md:290]:
```suggestion
  `Null`, it will also return `Null`.
```

File: docs/changelog.md [hunk 6]
```diff
@@ -239,31 +305,32 @@ sidebar_label: Changelog
...
 * `SHOW CONSTRAINT INFO` now returns property names as a list for unique
   constraints.
-* Escaped label/property/edgetype names in `DUMP DATABASE` to support names
-  with spaces in them.
-* Fixed handling of `DUMP DATABASE` queries in multi-command transactions
-  (`BEGIN`, ..., `COMMIT`).
+* Escaped label/property/edgetype names in `DUMP DATABASE` to support names with
+  spaces in them.
+* Fixed handling of `DUMP DATABASE` queries in multi-command transactions (
+  `BEGIN` , ..., `COMMIT` ).
```
Comment (kostasrim) [inline docs/changelog.md:328]: Are these extra spaces intentional ?

**Review by kostasrim — APPROVED** [review]

File: docs/changelog.md (hunk 6 above)
Comment (jbajic) [reply docs/changelog.md:328]: I am not sure why everything has been formatted...

File: docs/changelog.md (hunk 2 above)
Comment (jbajic) [reply docs/changelog.md]: its the first

File: docs/changelog.md (hunk 6 above)
Comment (kostasrim) [reply docs/changelog.md:328]: No worries!

**Review by kostasrim — APPROVED** [review]

**Pull request merged**